bin/create-app-secret

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+read -s -p "Enter the Discovery Django secret key (development): " discovery_django_secret_key
+echo
+if [ -z "${discovery_django_secret_key}" ]; then
+  discovery_django_secret_key="development"
```

Comment:
I see this file was renamed, but since this is the first time I see this snippet: I don't think we should be hardcoding django secret key (and why the message is "Enter the Discovery Django secret key (development)"? Production deployments do need this key as well...